README.md

```diff
@@ -85,3 +85,41 @@ Do the following:
    * `go get github.com/some/lib@v1.2.3`: Get a particular version.
 2. Run `go mod tidy` to remove any unused modules.
 3. Run `go mod verify` to add packages needed for test packages etc.
+
+## Pulling the docker image
+
+Unfortunately the Github packages docker repo is not very "public", more on this can be found in this community [thread](https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/td-p/32782/page/4). This means a docker login needs to be done before the images can be pulled. To do this follow these steps:
+
+1. Create a new token with the scope `read:packages` [here](https://github.com/settings/tokens).
+2. Save your token to e.g. to a file (or use environment variable or similar).
+3. Login with docker to `docker.pkg.github.com`.
+
+Using token saved to the file github.token: 
+
+```bash
+docker login -u yourgithubusername --password=$(cat github.token) docker.pkg.github.com
+```
+
+Using token in environmental variable GITHUB_TOKEN:
```

Comment:
Change ... "Using token in environmental variable"... to "Using a token in an environmental variable"...?

Reply:
"Using a token in the environmental variable GITHUB_TOKEN" better since its this specific one?

Reply:
Ah, sorry, maybe better with "Using the token in the environmental..."?